scripts/ci/run-local-chain-nettest.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2020 Tocqueville Group
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# This script runs given nettest scenario using local-chain with the node
+# on '127.0.0.1:8734' (currently used rpc address for local-chain jupiter node).
```

Comment:
It doesn't look right. There is no `127.0.0.1:8734` here, it comes from the outside. It should be fixed in `morley` as well btw.